server/app/registry.ts

```diff
@@ -36,6 +39,18 @@ logger.add(
   })
 );
 
+if (config.get<string>('zeetEnv').toLowerCase().includes('prod')) {
```

Comment:
zeetEnv will never contain `prod` in the case expected. zeetEnv is based on the branch that it is built off of, such as `develop` or `main`.

Reply:
so I put develop and main?

Reply:
Yeah, we'll log to LogDNA if the zeetEnv matches develop or main

Reply:
I think we should only log main, it will be confusing if not

Reply:
unless we can have two different views for the logs

Reply:
We can log when we are in main only

Reply:
Sure, main only works for me

Reply:
> unless we can have two different views for the logs

Yeah that's possible as well by create two "app" like "backend service develop" and "backend service main"

Reply:
so what do you guys want?

Reply:
I will put 'main' only

Reply:
What if one day we want to debug something from our develop branch using the logs?

I like the idea of having two views just in case

Reply:
but we are logging in the console

Reply:
> but we are logging in the console

When the backend is hosted on Zeet? Only Mack as access to the runtime logs. So if we log to LogDNA, everyone could have access to the runtime logs